### PR TITLE
Message category selection

### PR DESCRIPTION
**Pull request by drewbern**
Adds dropdown for posting to a specific category from the country page

### CODE REVIEW

File: src/main/webapp/WEB-INF/country.jsp [hunk 1]
```diff
@@ -35,15 +56,32 @@
...
     </div>
     <h1 id="page-title"><%= countryName %></h1>
 
+    <% if (currentUser != null) { %>
+    <form id="message-form" action="" method="POST" class>
+    Enter a new message:
+    <br/>
+    <textarea name="text" placeholder="Enter a message" id="message-input"></textarea>
+      <select id="myDropdown" onchange="updateMessageCategory()">
+        <!-- <option value="" selected disabled hidden>Select Category</option> -->
```
Comment (nbaldy) [inline src/main/webapp/WEB-INF/country.jsp]: Is this debugging stuff?

File: src/main/java/com/google/codeu/servlets/CategoryServlet.java [hunk 2]
```diff
@@ -3,6 +3,7 @@
 import com.google.codeu.data.Country;
 import com.google.codeu.data.Datastore;
 import java.io.IOException;
+import java.util.ArrayList;
```
Comment (nbaldy) [inline src/main/java/com/google/codeu/servlets/CategoryServlet.java]: Did you use the Arraylist anywhere in this? I don't see a change involving it

**Review by nbaldy — APPROVED** [review]
LGTM with a couple of questions below

File: src/main/webapp/WEB-INF/country.jsp (hunk 1 above)
Comment (drewbern) [reply src/main/webapp/WEB-INF/country.jsp]: It was a placeholder until the user selected a category value, but its tricky to deal with the user clicking submit without changing the category. So I just made the default category the General one.

File: src/main/java/com/google/codeu/servlets/CategoryServlet.java (hunk 2 above)
Comment (drewbern) [reply src/main/java/com/google/codeu/servlets/CategoryServlet.java]: No, I forgot to change that back, good catch!

**Pull request merged**